mandatory/3-playing-computer.js

```diff
@@ -17,20 +17,21 @@
 
 let x = 2;
 let a = 6;
+let b = 3;
 
 const f1 = function (a, b) {
   return a + b;
 };
 
-const f2 = function (a, b) {
+const f2 = function (a, b, x) {
   return a + b + x;
 };
 
 console.log(x);
 console.log(a);
 console.log(b);
 
-for (let i = 0; i < 5; ++i) {
+for (let i = 0; i < 5; i++) {
   a = a + 1;
   if (i % 2 === 0) {
     const d = f2(i, x);
```

Comment:
Don't you think f2 needs three arguments?

Reply:
Thank you Mohammadreza,  and but on the second comment why 3 arguments on it.